roborazzi-idea-plugin/src/main/kotlin/com/github/takahirom/roborazzi/idea/preview/RoborazziPreviewTool.kt

```diff
@@ -241,6 +241,56 @@ class ImageListCellRenderer : ListCellRenderer<Pair<String, Long>> {
     val isSelected: Boolean
   )
 
+  private fun collapseFileNameToFitWidth(text: String, maxWidth: Int): String {
+    val label = JBLabel(text)
+    val fontMetrics = label.getFontMetrics(label.font)
+
+    // Adjust the maximum width to take into account the width of the scrollbar (about 20 pixels).
+    val bufferedWidth = maxWidth - 20
```

Comment:
Thanks. I'm not sure if this is related to this pull request, but my IDE froze when I was shrinking the tool window.
<img width="234" alt="image" src="https://github.com/user-attachments/assets/12e0d201-80a9-436e-87af-b97318c493b6">

Reply:
Hmm...
Excuse me, is this a screenshot test of the Doroidkaigi2024 project?
In my environment, it's  `./gradlew runIde` to start intellij and select `conference-app-2024`, I get an AGP compatibility error.
Are you debugging the Roborazzi plugin in some other way?

Reply:
Thanks!
I could reproduce it.
You are right, the “-20” could be doing something wrong...🤔
I'll try to make some more improvements.